spec/controllers/glossaries_controller_spec.rb

```diff
@@ -167,4 +149,26 @@
     end
   end
 
+  describe "duplicate" do
+    it "duplicates the requested glossary" do
+      expect {
+        post :duplicate, {:id => glossary.id}
```

Comment:
switch to get

Reply:
Fixed in https://github.com/concord-consortium/lara/pull/931